src/alloc.c

```diff
@@ -308,6 +308,22 @@ DLLEXPORT jl_value_t *jl_new_struct_uninit(jl_datatype_t *type)
     return jv;
 }
 
+DLLEXPORT jl_bytevec_struct_t jl_bytevec(const uint8_t *data, size_t n)
```

Comment:
it's weird because it's got a gc pointer it in, but it's invisible to the gc when written this way